auomsctl.cpp

```diff
@@ -1095,16 +1096,35 @@ int set_rules() {
     }
 
     try {
-        auto rules = ReadActualAuditdRules(false, nullptr);
-        auto diff = DiffRules(rules, desired_rules, "");
+        std::vector<std::string> errors;
+        auto rules = ReadActualAuditdRules(false, &errors);
+        auto merged_rules = MergeRules(rules);
+        auto diff = DiffRules(merged_rules, desired_rules, "");
+        if (!errors.empty()) {
```

Comment:
>if (!errors.empty()) { [](start = 8, length = 22)

Please consider moving this check on errors before calling MergeRules() and just after ReadActualAuditdRules()

Reply:
Done.